sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala

```diff
@@ -83,7 +83,10 @@ package object dsl {
     def >= (other: Expression): Predicate = GreaterThanOrEqual(expr, other)
     def === (other: Expression): Predicate = EqualTo(expr, other)
     def <=> (other: Expression): Predicate = EqualNullSafe(expr, other)
```

Comment:
@jodersky Hi Jakob, <=> is a null safe equal operator.  Here is a link that describes the semantics.
http://dev.mysql.com/doc/refman/5.0/en/comparison-operators.html#operator_equal-to